packages/pkp-base/src/lib/pkp-base.ts

```diff
@@ -103,7 +103,7 @@ export class PKPBase<T = PKPBaseDefaultParams> {
       minNodeCount:
         prop.bootstrapUrls && prop.litNetwork == 'custom'
           ? prop.minNodeCount
-          : 6,
+          : 2,
```

Comment:
Can this be read from the network config instead of hard coded? we then dont have to update per network

Reply:
https://github.com/LIT-Protocol/js-sdk/blob/952f7abecdafc451599c6734e6a6a3c2bb0b975b/packages/pkp-base/src/lib/pkp-base.ts#L107